services/app-api/src/testHelpers/gqlHelpers.ts

```diff
@@ -178,6 +178,40 @@ const updateTestHealthPlanFormData = async (
     return updateResult.data.updateHealthPlanFormData.pkg
 }
 
+const updateTestHealthPlanPackage = async (
```

Comment:
I'm confused, this function seems like it already exists on main?

Reply:
@macrael  I made because the function on main  `updateTestHealthPlanFormData` takes in complete form data, proto-izes it and returns it. 

I made a `updateTestHealthPlanPackage` to make an update call that is more similar to the other test gql helpers at this point in our code base -  it takes in partial updates and conditionally applies them but otherwise self generates its own faked data.